src/purify.ts

```diff
@@ -819,33 +822,34 @@ function createDOMPurify(window: WindowLike = getGlobal()) {
   /**
    * _removeAttribute
    *
-   * @param  {String} name an Attribute name
-   * @param  {Node} node a DOM node
+   * @param  {string} name an Attribute name
+   * @param  {Element} element a DOM node
+   * @returns {void}
    */
-  const _removeAttribute = function (name, node) {
+  const _removeAttribute = function (name: string, element: Element): void {
```

Comment:
I set the `Node` type to `Element` because it gives a type error, and rename it accordingly.